### ISSUE TITLE
Minor: config show output shows opposite of whether or not it's the default configuration

### ISSUE THREAD

**Issue by wmoy-thoughtspot** [thoughtspot/cs_tools#112]
### First Stop

- [X] I have read and agreed to the ThoughtSpot Dev Tools EULA.
- [X] I understand that if I have a critical issue with ThoughtSpot, I should reach out to my designated Account Executive and Solutions Consultant team instead of cs_tools.
- [X] I searched the cs_tools documentation, with the integrated search.

### Platform Configuration

```
╭──────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│                                                                                                                  │
│           Info snapshot taken on 2024-03-14                                                                      │
│                                                                                                                  │
│               CS Tools: 1.4.15                                                                                   │
│         Python Version: Python 3.9.4 (tags/v3.9.4:1f2e308, Apr  6 2021, 13:40:21) [MSC v.1928 64 bit (AMD64)]    │
│            System Info: Windows (detail: Windows-10-10.0.22621-SP0)                                              │
│      Configs Directory: C:\Users\ {anonymous} \AppData\Roaming\cs_tools                                          │
│    Activate VirtualEnv: C:\Users\ {anonymous} \AppData\Roaming\cs_tools\.cs_tools\Scripts\Activate.ps1           │
│          Platform Tags: win-amd64                                                                                │
│                                                                                                                  │
╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
```

### Description

When running cs_tools config show --config, the output is stating incorrect (opposite?) designation of whether a particular configuration is the default or not.  Attached screenshot shows installation with 3 configs and that "cantire_prod" is the default but the output of config show for each of them shows the opposite.

![image](...)

**Comment by boonhapus** [comment]
Gotcha , thanks @wmoy-thoughtspot ! I've removed the image you attached but I can see where this is going wrong.

**Comment by boonhapus** [comment]
This has been fixed in 1.5.0 😄
